<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customerProfile</class>
 <widget class="QDialog" name="customerProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>458</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>694</width>
    <height>458</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>694</width>
    <height>458</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AvisionR - Client Profile</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:rgb(255, 255, 255);
background-color: #262626;</string>
  </property>
  <widget class="QLabel" name="label_addressLine2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Address Line 2</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_History">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>671</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QHeaderView::section {
background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                       stop:0 #616161, stop: 0.5 #505050,
                                       stop: 0.6 #434343, stop:1 #656565);
     color: white;
     padding-left: 4px;
     border: 1px solid #6c6c6c;

}

QTableCornerButton::section 
{
	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                       stop:0 #616161, stop: 0.5 #505050,
                                       stop: 0.6 #434343, stop:1 #656565);
     color: white;
     padding-left: 4px;
     border: 1px solid #6c6c6c;
}

color:rgb(255, 255, 255);
background-color: #262626;
</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton_newPurchase">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>410</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>New Purchase</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>410</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Modify Profile</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton_3">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>410</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>History:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_phoneNumber">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_addressLine1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>41</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Address Line 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Fullname">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>111</width>
     <height>16</height>
    </size>
   </property>
   <property name="text">
    <string>Full Name       </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>551</x>
     <y>41</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Balance: $</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_balance">
   <property name="geometry">
    <rect>
     <x>607</x>
     <y>41</y>
     <width>77</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Balance Amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>551</x>
     <y>21</y>
     <width>43</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>43</width>
     <height>13</height>
    </size>
   </property>
   <property name="text">
    <string>Account:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_account">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>21</y>
     <width>79</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>79</width>
     <height>13</height>
    </size>
   </property>
   <property name="text">
    <string>Account Number</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
